app/frontend/src/features/communities/CommunityPage/PlaceCard.tsx

```diff
@@ -0,0 +1,65 @@
+import {
+  Card,
+  CardActionArea,
+  CardContent,
+  CardMedia,
+  makeStyles,
+  Typography,
+} from "@material-ui/core";
+import classNames from "classnames";
+import React, { useMemo } from "react";
+import LinesEllipsis from "react-lines-ellipsis";
+
+import { Page } from "../../../pb/pages_pb";
+import stripMarkdown from "../../../utils/stripMarkdown";
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    maxWidth: 160,
```

Comment:
Since the text gets truncated, wouldn't we want to set the `width` instead so they all have the same width?

Although not sure about having magic numbers like this here. Feel like it's probably better to let the parent component that will presumably render these cards in a list to decide its width depending on different screen sizes etc.